README.md

```diff
@@ -8,12 +8,9 @@
 
 ---
 
-Please note that this repository is participating in a study into sustainability
-of open source projects. Data will be gathered about this repository for
-approximately the next 12 months, starting from 2021-06-11.
+Please note that this repository is participating in a study into the sustainability of open source projects. Data will be gathered about this repository for approximately the next 12 months, starting from 2021-06-11.
 
-Data collected will include number of contributors, number of PRs, time taken to
-close/merge these PRs, and issues closed.
+Data collected will include a number of contributors, number of PRs, time taken to close/merge these PRs, and issues closed.
```

Comment:
```suggestion
Data collected will include number of contributors, number of PRs, time taken to close/merge these PRs, and issues closed.
```
I think the original sentence was fine, alternatively you could say ***the*** *number of contributors*?

Reply:
Thank you for the suggestion! You are right.